topics/micro/index.md

```diff
@@ -0,0 +1,11 @@
+---
+display_name: Micro
+topic: micro
```

Comment:
Unfortunately, this is quite a ubiquitous name on GitHub. Even the target topic page's top repository is unrelated to github.com/micro. Can you think of any other topic names that would be appropriate and more targeted? 

perhaps:
```suggestion
topic: micro-platform
```

or:
```suggestion
topic: m3o
```

Reply:
Thanks for your feedback @kenyonj.

As far as I know, *Micro* is the name of the organization, the actual project and even the company behind it; so I'm not sure it would make much sense to have any other name than that as topic name.

Please let me know if there's any other issue to be fixed. Appreciate your time.

Reply:
> Please let me know if there's any other issue to be fixed.

No other issues besides the topic name, once that’s changed we can get this merged in. The fact that it’s the organization’s name doesn’t change the fact that it’s an established topic tag and we can’t brand it without causing confusion.